### PR TITLE
Add datenlord metrics exporter

### PR DESCRIPTION
**Pull request by mrwwheat7**

### CODE REVIEW

File: async_fuse/src/main.rs [hunk 1]
```diff
@@ -284,6 +293,17 @@ fn main() -> anyhow::Result<()> {
         None => CACHE_DEFAULT_CAPACITY,
     };
 
+    smol::spawn(Compat::new(async move {
```
Comment (rogercloud) [inline async_fuse/src/main.rs:296]: Please move this part to the metrics internal so that the metrics service only takes one line in the main function.

**Review by rogercloud — CHANGES_REQUESTED** [review]
So your plan is to add the tests in the next commit?

File: async_fuse/src/main.rs (hunk 1 above)
Comment (mrwwheat7) [reply async_fuse/src/main.rs]: DONE.

**Comment by mrwwheat7** [comment]
> So your plan is to add the tests in the next commit?

Yes, the test is in the next PR

File: async_fuse/src/main.rs [hunk 2]
```diff
@@ -193,6 +196,7 @@ async fn register_volume(
     Ok(())
 }
 
+#[allow(clippy::too_many_lines)] //allow this for argument parser function as there is no other logic in this function
```
Comment (rogercloud) [inline async_fuse/src/main.rs:199]: can be removed

**Review by rogercloud — APPROVED** [review]

File: async_fuse/src/main.rs (hunk 2 above)
Comment (mrwwheat7) [reply async_fuse/src/main.rs]: Done.

**Review by rogercloud — APPROVED** [review]

**Pull request merged**